website/pages/en/community.js

```diff
@@ -177,6 +176,26 @@ function Community(props) {
     </CommunityResource>
   );
 
+  const Events = () => (
+    <CommunityResource title={<translate>Events & Meetups</translate>}>
+      <p>
+        <translate>
+          Find recent events and meetups that you love and get together to talk with other developers and researchers.
```

Comment:
"Find upcoming events where you can get together and talk with other Substrate developers."